exs/mundo_1/c#/023.cs

```diff
@@ -0,0 +1,42 @@
+/*
+Desafio 023
+
+Problema: Faça um programa que leia um número de 0 a 9999 e mostre na tela 
+          cada um dos dígitos separados.
+
+Resolução do problema:
+*/
+
+using System;
+
+class io {
+    public static string input(string text) {
+        Console.Write(text);
+        return Console.ReadLine();
+    }
+    
+    public static void print(object data, string end="\n") {
+        string text = data.ToString();
+
+        Console.Write($"{text}{end}");
+    }
+}
+
+
+class Program {
+  public static void Main () {
+    int num = int.Parse(io.input(
+      "Insira um número de 1 a 9999: "
```

Comment:
O range correto é de **0 a 9999**, conforme descrito no enunciado

```suggestion
      "Insira um número de 0 a 9999: "
```